Services/Saml/classes/class.ilSamlAttributesHolder.php

```diff
@@ -0,0 +1,55 @@
+<?php
+// saml-patch: begin
+/* Copyright (c) 1998-2016 ILIAS open source, Extended GPL, see docs/LICENSE */
+
+/**
+ * Class ilAuthContainerSAML
+ * This is a helper class to transport the simplesamlphp attributes to the auth container.
+ * This is indeed also global scope, but better than a global array.
+ * @author Michael Jansen <mjansen@databay.de>
+ */
+class ilSamlAttributesHolder
```

Comment:
I will try to get rid of this class. It was introduced during the initial SAML impl. (done for ILIAS 4.4.x) and used as a global transport mechanism for the SAML attributes. Maybe this is not necessary with the revised >= 5.2.x auth. code anymore.